### PR TITLE
Updated travis.yml and created appropriate test files

### LINKED ISSUE #517: Integrate WDL testing into the automated tests
We should run the case and PoN creation wdl workflows when doing automated testing.  

Depends on #516

### PR DESCRIPTION
**Pull request by asmirnov239**
Closes #517 Updated travis.yml file and added test files for testing ACNV case and PoN pipelines, including downsampled WGS and WES bams, truncated reference, target and common SNPs files. Put all the necessary scripts in scripts/acnv_cromwell_tests directory

### CODE REVIEW

**Comment by asmirnov239** [comment]
@samuelklee Could you please take a look at this? thank you!

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "case_gatk_acnv_workflow.ref_fasta_dict": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict",
+  "case_gatk_acnv_workflow.input_bam_list": "/home/travis/build/broadinstitute/gatk-protected/scripts/acnv_cromwell_tests/case_input_exome.tsv",
+  "case_gatk_acnv_workflow.CalculateTargetCoverage.keepduplicatereads": "true",
+  "case_gatk_acnv_workflow.PoN": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/exome_test.pon",
+  "case_gatk_acnv_workflow.CalculateTargetCoverage.transform": "PCOV",
+  "case_gatk_acnv_workflow.ref_fasta": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta",
+  "case_gatk_acnv_workflow.NormalizeSomaticReadCounts.mem": "4",
+  "case_gatk_acnv_workflow.PadTargets.pd": "250",
+  "case_gatk_acnv_workflow.disable_sequence_dictionary_validation": "true", 
+  "case_gatk_acnv_workflow.plots_dir": "/tmp/asmirnov/plots/",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json]: can you change all tmp directory names that contain "asmirnov"?

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "case_gatk_acnv_workflow.ref_fasta_dict": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict",
+  "case_gatk_acnv_workflow.input_bam_list": "/home/travis/build/broadinstitute/gatk-protected/scripts/acnv_cromwell_tests/case_input_wgs.tsv",
+  "case_gatk_acnv_workflow.CalculateTargetCoverage.keepduplicatereads": "true",
+  "case_gatk_acnv_workflow.PoN": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/wgs_test.pon",
+  "case_gatk_acnv_workflow.CalculateTargetCoverage.transform": "PCOV",
+  "case_gatk_acnv_workflow.ref_fasta": "/home/travis/build/broadinstitute/gatk-protected/src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.fasta",
+  "case_gatk_acnv_workflow.NormalizeSomaticReadCounts.mem": "4",
+  "case_gatk_acnv_workflow.PadTargets.pd": "250",
+  "case_gatk_acnv_workflow.disable_sequence_dictionary_validation": "true",
+  "case_gatk_acnv_workflow.plots_dir": "/tmp/asmirnov/plots/",
```
Comment (samuelklee) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json]: same here

File: src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict [hunk 3]
```diff
@@ -0,0 +1,2 @@
+@HD	VN:1.5	SO:unsorted
+@SQ	SN:20	LN:1000000	M5:e7203581340e18505b7b4f24978be166	UR:file:/dsde/working/asmirnov/TestCromwell/WGS_scripts/human_g1k_v37.20.21-truncated.fasta
```
Comment (samuelklee) [inline src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict]: you can also edit the UR tag if you want to replace with some dummy file path

File: src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.txt [hunk 4]
```diff
@@ -0,0 +1,94 @@
+contig	start	stop	name
```
Comment (samuelklee) [inline src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.txt]: can you change the extension of this file to tsv?

**Comment by samuelklee** [comment]
looks good!  just a few very minor changes, then feel free to squash and merge.

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json (hunk 1 above)
Comment (asmirnov239) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_exome.json]: done

File: scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json (hunk 2 above)
Comment (asmirnov239) [inline scripts/acnv_cromwell_tests/case_gatk_acnv_workflow_wgs.json]: done

File: src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict (hunk 3 above)
Comment (asmirnov239) [inline src/test/resources/large/ACNV_pipeline_test_files/human_g1k_v37.chr-20.truncated.dict]: done

File: src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.txt (hunk 4 above)
Comment (asmirnov239) [inline src/test/resources/large/ACNV_pipeline_test_files/ice_targets_sample-chr20.txt]: done

**Pull request merged**